applications/schema/inputs.py

```diff
@@ -40,8 +40,8 @@ class BaseApplicationInput(graphene.InputObjectType):
 
 
 class BerthApplicationInput(BaseApplicationInput):
-    boat_draught = graphene.Float()
-    boat_weight = graphene.Float()
+    boat_draught = graphene.Decimal()
```

Comment:
FYI: Tested this with berth-reservation-admin (UI) `yarn type:generator`. The numbers are changed to now with the any-type. Floats were typed as numbers.